src/Memory/ILogStorage.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import * as CLM from '@conversationlearner/models'
+
+export interface LogQueryResult {
```

Comment:
>LogQueryResult [](start = 17, length = 14)

you can reuse LogDialogList here and add the nullable continuationToken there. Or you can define a PagedQueryResult<T> nad ILogStorage methods return PagedQueryResut<LogDialog>.